OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
 		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
 		public class Consideration
 		{
-			public enum DecisionMetric { Health, Value, None }
+			public enum DecisionMetric { Health, Value, BaseBuilding, None }
```

Comment:
Please point out which file have those target types definition in the rules
folder

On Thu, Nov 12, 2015 at 4:47 PM, Paul Chote <notifications@github.com>
wrote:

> In OpenRA.Mods.Common/AI/SupportPowerDecision.cs
> <https://github.com/OpenRA/OpenRA/pull/9936#discussion_r44664522>:
>
> > @@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
> >  		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
> >  		public class Consideration
> >  		{
> > -			public enum DecisionMetric { Health, Value, None }
> > +			public enum DecisionMetric { Health, Value, BaseBuilding, None }
>
> You can add a new TargetType to construction yards, and then reference
> that in the Types list. They are all defined in the mod rules. @UberWaffe
> <https://github.com/UberWaffe> is correct that this doesn't require any
> new code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenRA/OpenRA/pull/9936/files#r44664522>.
>



-- 
Best Regards,
Fady Anwar